plenum/test/pp_seq_no_restoration/test_node_erases_last_sent_pp_key_on_propagate_primary_after_pool_restart.py

```diff
@@ -1,108 +0,0 @@
-from plenum.common.constants import LAST_SENT_PRE_PREPARE
-from plenum.test import waits
-from plenum.test.checkpoints.conftest import chkFreqPatched
-from plenum.test.helper import sdk_send_batches_of_random, assertExp
-from plenum.test.pool_transactions.helper import \
-    disconnect_node_and_ensure_disconnected
-from plenum.test.test_node import ensureElectionsDone, getPrimaryReplica, \
-    checkNodesConnected
-from plenum.test.view_change.helper import start_stopped_node
-from stp_core.loop.eventually import eventually
-
-CHK_FREQ = 2
-LOG_SIZE = 3 * CHK_FREQ
-
-nodeCount = 4
-
-backup_inst_id = 1
-
-
```

Comment:
Why did we remove this test?

Reply:
pool restart will not erase last 3pc on master anymore, because of audit ledger, so we probably don't need to check this case